src/uct/ib/rc/accel/rc_mlx5.h

```diff
@@ -96,19 +96,21 @@ ssize_t uct_rc_mlx5_base_ep_put_bcopy(uct_ep_h tl_ep,
                                       void *arg, uint64_t remote_addr,
                                       uct_rkey_t rkey);
 
-ucs_status_t uct_rc_mlx5_ep_put_zcopy(uct_ep_h tl_ep, const uct_iov_t *iov, size_t iovcnt,
-                                      uint64_t remote_addr, uct_rkey_t rkey,
-                                      uct_completion_t *comp);
+ucs_status_t uct_rc_mlx5_base_ep_put_zcopy(uct_ep_h tl_ep, const uct_iov_t *iov,
```

Comment:
seems all renamed now except `uct_rc_mlx5_ep_put_short`. Is there a reson for that?

Reply:
fixed